apollo-ios-codegen/Sources/GraphQLCompiler/JavaScript/src/utilities/apolloCodegenSchemaExtension.ts

```diff
@@ -9,6 +9,22 @@ export const directive_apollo_client_ios_localCacheMutation: DirectiveDefinition
   locations: [nameNode("QUERY"), nameNode("MUTATION"), nameNode("SUBSCRIPTION"), nameNode("FRAGMENT_DEFINITION")]
 }
 
+export const directive_typePolicy: DirectiveDefinitionNode = {
```

Comment:
For the record, the directives are specified [here](https://specs.apollo.dev/kotlin_labs/v0.4/). We might want to change the name of the spec now :)